neat_internal.h

```diff
@@ -21,6 +21,7 @@
 #endif
 
 #include "neat_log.h"
+#include "neat_stat.h"
```

Comment:
Why include neat_stat.h here, there don't seem to be any further changes in this header requiring anything from there?